tests/jvm/src/test/resources/example/Synthetic_2.13.8.scala

```diff
@@ -1,22 +1,22 @@
-  Array/*=>scala.Array.*/.empty/*=>scala.Array.empty().*/[Int/*=>scala.Int#*/].headOption/*=>scala.collection.TraversableLike#headOption().*/
-  "fooo".stripPrefix/*=>scala.collection.immutable.StringLike#stripPrefix().*/("o")
+  List/*=>scala.package.List.*/(1).map/*=>scala.collection.immutable.List#map().*/(_ +/*=>scala.Int#`+`(+4).*/ 2)
+  Array/*=>scala.Array.*/.empty/*=>scala.Array.empty().*/[Int/*=>scala.Int#*/].headOption/*=>scala.collection.ArrayOps#headOption().*/
+  "fooo".stripPrefix/*=>scala.collection.StringOps#stripPrefix().*/("o")
 
   // See https://github.com/scalameta/scalameta/issues/977
-  val Name/*<=example.Synthetic#Name.*/ = "name:(.*)".r/*=>scala.collection.immutable.StringLike#r().*/
+  val Name/*<=example.Synthetic#Name.*/ = "name:(.*)".r/*=>scala.collection.StringOps#r().*/
   val x/*<=example.Synthetic#x.*/ #::/*=>scala.package.`#::`.*/ xs/*<=example.Synthetic#xs.*/ = Stream/*=>scala.package.Stream.*/(1, 2)
   val Name/*=>example.Synthetic#Name.*/(name/*<=example.Synthetic#name.*/) = "name:foo"
-  1 #::/*=>scala.collection.immutable.Stream.ConsWrapper#`#::`().*/ 2 #::/*=>scala.collection.immutable.Stream.ConsWrapper#`#::`().*/ Stream/*=>scala.package.Stream.*/.empty/*=>scala.collection.immutable.Stream.empty().*/
+  1 #::/*=>scala.collection.immutable.Stream.Deferrer#`#::`().*/ 2 #::/*=>scala.collection.immutable.Stream.Deferrer#`#::`().*/ Stream/*=>scala.package.Stream.*/.empty/*=>scala.collection.immutable.Stream.empty().*/
 
-  val lst/*<=example.Synthetic#lst.*/ = 1 #::/*=>scala.collection.immutable.Stream.ConsWrapper#`#::`().*/ 2 #::/*=>scala.collection.immutable.Stream.ConsWrapper#`#::`().*/ Stream/*=>scala.package.Stream.*/.empty/*=>scala.collection.immutable.Stream.empty().*/
+  val lst/*<=example.Synthetic#lst.*/ = 1 #::/*=>scala.collection.immutable.Stream.Deferrer#`#::`().*/ 2 #::/*=>scala.collection.immutable.Stream.Deferrer#`#::`().*/ Stream/*=>scala.package.Stream.*/.empty/*=>scala.collection.immutable.Stream.empty().*/
   lst/*=>example.Synthetic#lst.*/ +/*=>scala.Predef.any2stringadd#`+`().*/ "foo"
 
-  for (x/*<=local7*/ <- 1 to/*=>scala.runtime.RichInt#to().*/ 10; y/*<=local8*/ <- 0 until/*=>scala.runtime.RichInt#until().*/ 10) println/*=>scala.Predef.println(+1).*/(x/*=>local7*/ ->/*=>scala.Predef.ArrowAssoc#`->`().*/ x/*=>local7*/)
-  for (i/*<=local9*/ <- 1 to/*=>scala.runtime.RichInt#to().*/ 10; j/*<=local10*/ <- 0 until/*=>scala.runtime.RichInt#until().*/ 10) yield (i/*=>local9*/, j/*=>local10*/)
-  for (i/*<=local11*/ <- 1 to/*=>scala.runtime.RichInt#to().*/ 10; j/*<=local12*/ <- 0 until/*=>scala.runtime.RichInt#until().*/ 10 if i/*=>local11*/ %/*=>scala.Int#`%`(+3).*/ 2 ==/*=>scala.Int#`==`(+3).*/ 0) yield (i/*=>local11*/, j/*=>local12*/)
+  for (x/*<=local3*/ <- 1 to/*=>scala.runtime.RichInt#to().*/ 10; y/*<=local4*/ <- 0 until/*=>scala.runtime.RichInt#until().*/ 10) println/*=>scala.Predef.println(+1).*/(x/*=>local3*/ ->/*=>scala.Predef.ArrowAssoc#`->`().*/ x/*=>local3*/)
```

Comment:
and why are locals here numbered differently?

Reply:
Might be a difference in the underlying implementation of for comprehension, but not sure. As long as the changes are consistent it's fine. Looks like we are just missing a couple of locals that were not even shown at all in occurrences.